features/steps/run_steps.py

```diff
@@ -51,7 +51,7 @@ def create_project_from_config_file(context, starter_name):
 def install_project_dependencies(context):
     reqs_path = "src/requirements.txt"
     res = subprocess.run(
-        [context.pip, "install", "-r", reqs_path], cwd=context.root_project_dir
+        [context.pip, "install", "-r", reqs_path, "-U"], cwd=context.root_project_dir
```

Comment:
`-U` is `upgrade`. You'll need `--user` for the permission stuff. (see https://pip.pypa.io/en/stable/cli/pip_install/)

Reply:
I don't think we can use `--user` since we use a [virtual environment](https://bobbyhadz.com/blog/python-could-not-install-packages-due-to-an-environmenterror-winerror-5#:~:text=However%2C%20the%20%2D%2Duser%20option%20wouldn%27t%20work%20if%20you%20have%20a%20virtual%20environment%20active.).

Reply:
@merelcht No luck just yet, unfortunately. `--user` doesn't work because of the virtual environment as @SajidAlamQB pointed out. My theory is that the error is due to a process lock, see [here](https://stackoverflow.com/questions/51912999/could-not-install-packages-due-to-an-environmenterror-winerror-5-access-is-de):
> Turns out, Windows gives permission error when the file is locked by a process. Python reports it as [Winerror 5] and I could not easily find that documentation reference anyway. 

A workaround is still being searched for, any ideas are welcome 😄 

Whilst the error is not fixed, I do believe we should have been using the upgrade flag.

Additionally, the flag is now allowing for the 3.10 Windows tests to pass. Still investigating as to why.